src/openrct2/world/Location.hpp

```diff
@@ -162,6 +162,27 @@ struct CoordsXY
     };
 };
 
+struct CoordsXYRangedZ : public CoordsXY
+{
+    int32_t zBegin = 0;
+    int32_t zEnd = 0;
+
+    CoordsXYRangedZ() = default;
+    constexpr CoordsXYRangedZ(int32_t _x, int32_t _y, int32_t _zBegin, int32_t _zEnd)
+        : CoordsXY(_x, _y)
+        , zBegin(_zBegin)
```

Comment:
zLow

Reply:
I've gone with `baseZ` and `clearanceZ`, given the recent PR you did, if people think otherwise, I can change, it's not a big one.